neqo-transport/src/connection.rs

```diff
@@ -2359,6 +2554,7 @@ mod tests {
             Rc::new(RefCell::new(FixedConnectionIdManager::new(3))),
             loopback(),
             loopback(),
+            QuicVersion::default(),
```

Comment:
I still don't see any -27-specific tests.

Reply:
You're not getting them in this PR, that's what #692 is for.